CHANGELOG.md

```diff
@@ -77,6 +78,10 @@
 - Added support for `DataFrame.corr`.
 - Added support for `DataFrame.equals` and `Series.equals`.
 - Added support for `DataFrame.reindex` and `Series.reindex`.
+- Added support for `Index.astype`.
```

Comment:
Is this related to this pr?  I don't see any code changes related to 'Index.astype'

Reply:
@sfc-gh-evandenberg it is not related, it is a fix of the previous wrong change log conflict resolving. @sfc-gh-vbudati can you actually put that in a different pr, we can merge it separately

Reply:
I'll put this in a different PR, here: https://github.com/snowflakedb/snowpark-python/pull/1939